src/packages/shared-types/forms.ts

```diff
@@ -111,12 +109,15 @@ export type RHFComponentMap = {
   };
   FieldArray: {
     appendText?: string;
+    fieldArrayClassName?: string;
   };
   FieldGroup: {
     appendText?: string;
     removeText?: string;
+    fieldArrayClassName?: string;
   };
```

Comment:
Using fieldGroup has some base styling, as well as not having to go back and change all the places it was used in those other forms. But it could be removed if we want.

Reply:
ie. Button styling, base layout, and an additional text option for removing groups. All possible to change/remove, but where my hesitancy lied.